lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -373,50 +373,25 @@ export abstract class ClientApplication {
      * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      *
      */
-    async acquireTokenSilent(request: SilentRequest): Promise<AuthenticationResult> {
+    protected async refreshToken(request: SilentRequest): Promise<AuthenticationResult> {
```

Comment:
change the name of this to `acquireTokenByRefreshToken`

Reply:
+1 Can be confusing otherwise!

Reply:
Done